Accordion.js

```diff
@@ -0,0 +1,496 @@
+/** @module deliteful/Accordion */
+define(["dcl/dcl",
+	"decor/sniff",
+	"requirejs-dplugins/Promise!",
+	"delite/register",
+	"delite/KeyNav",
+	"requirejs-dplugins/jquery!attributes/classes",
+	"delite/DisplayContainer",
+	"./ToggleButton",
+	"delite/theme!./Accordion/themes/{{theme}}/Accordion.css"
+], function (dcl, has, Promise, register, KeyNav, $, DisplayContainer, ToggleButton) {
+
+	function setVisibility(node, val) {
+		node.style.display = val ? "" : "none";
+	}
+
+	var animationEndEvent = (function () {
```

Comment:
I agree... well this code is already doing feature testing, it's just that it's got a strange format of

```
var animationEndEvent = (function () { ... }());
```

instead of

```
has.add("animationEndEvent", function () { ... });
```